docs/develop/develop-overview.md

```diff
@@ -33,6 +33,9 @@ Create your project using HTML, CSS and JavaScript (or TypeScript), or using Rea
 
 ### Visual Studio
 
+> [!NOTE]
+> Beginning with Visual Studio 2026, Office Add-in development in Visual Studio is deprecated and will be removed in a future release.
+
 Visual Studio can be used to create Office Add-ins for Excel, Outlook, Word, and PowerPoint. An Office Add-in project gets created as part of a Visual Studio solution and uses HTML, CSS, and JavaScript. For more information about creating add-ins with Visual Studio, see [Develop Office Add-ins with Visual Studio](../develop/develop-add-ins-visual-studio.md).
```

Comment:
Can we use the text "We recommend creating Office Add-in projects with the Microsoft 365 Agents Toolkit or the Yeoman Generator." Each of those can link to the correct article like https://learn.microsoft.com/en-us/office/dev/add-ins/develop/yeoman-generator-overview